### DISCUSSION TITLE
Export private key

### DISCUSSION THREAD

**Discussion by kamilgrzela** [polhenarejos/pico-hsm#5] [Q&A] [answered]
Hi all,
First of all, thank you for the good project and concept. I am testing it right now. We will see what the result will be. 
And I have a question - Could you tell me why you are using the private key export option? 

BR

**Comment by polhenarejos** [comment, marked as answer]
You have two ways of exporting:
- PKCS12 format is used, for instance, to perform S/MIME signing. Most of email clients accept PKCS12 cert+privkey file formats. In this case, the private key is protected with a passphrase you provide.
- Wrap/unwrap is used to backup your private keys. It is useful to export them to another device or just for having a backup in case  your device suddenly deads. In this approach, the private key is wrapped by using a device key, usually generated from multiple shares custodied by different people.

Besides of that, you can mark a key as non-extractable and, therefore, impossible to backup.
